File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnDtoLayoutFormContentItem.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnShowBorderKind;
import com.neome.java.api.meta.base.Types.EnumDefnThemeDividerKind;
import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdGrid;
import com.neome.java.api.meta.base.Types.MetaIdLayoutForm;
import com.neome.java.api.meta.base.Types.MetaIdLayoutGrid;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnDtoLayoutFormContentItem
{
  @Nullable
  public DefnDtoColor borderColorVar;

  @Nullable
  public EnumDefnShowBorderKind[] borderPositionSet;

  @Nullable
  public MetaIdField[] fieldIdSet;

  @Nullable
  public MetaIdLayoutForm[] formLayoutIdSet;

  @Nullable
  public MetaIdLayoutGrid[] gridLayoutIdSet;

  @Nullable
  public EnumDefnShowBorderKind[] paddingPositionSet;

  @Nullable
  public EnumDefnThemeDividerKind paddingSize;

  @Nullable
  public MetaIdGrid[] showGridSwitcher;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnDtoLayoutFormContentItem.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnDtoColor
import com.neome.api.meta.base.Types.EnumDefnShowBorderKind
import com.neome.api.meta.base.Types.EnumDefnThemeDividerKind
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdGrid
import com.neome.api.meta.base.Types.MetaIdLayoutForm
import com.neome.api.meta.base.Types.MetaIdLayoutGrid

interface DefnDtoLayoutFormContentItem
{
  val borderColorVar: DefnDtoColor?
  val borderPositionSet: List<EnumDefnShowBorderKind>?
  val fieldIdSet: List<MetaIdField>?
  val formLayoutIdSet: List<MetaIdLayoutForm>?
  val gridLayoutIdSet: List<MetaIdLayoutGrid>?
  val paddingPositionSet: List<EnumDefnShowBorderKind>?
  val paddingSize: EnumDefnThemeDividerKind?
  val showGridSwitcher: List<MetaIdGrid>?
}